docs/how_to_guides/create_and_edit_expectations/how_to_create_expectations_between_tables.rst

```diff
@@ -0,0 +1,44 @@
+.. _tutorial_create_expectations_between_tables:
+
+Create Expectations Between Tables
+===================================
+
+This tutorial covers the workflow of creating and editing expectations that relate to data stored in different tables or datasets.
+
+Unfortunately, no Expectations natively support cross-table comparisons today (but we hope to in the future!). Consequently, there are two available paths:
+
+1. Use "Check Assets", where you create a new table that joins the two tables; or
+2. Use :ref:`Evaluation Parameters`_ to supply relevant metrics to expectations.
+
+
+Build a Check Asset
```

Comment:
There's a check asset article in the "spare parts" section that might be relevant to link to or combine with this.

Reply:
Gah! Great call. I didn't actually realize that existed. Since this branch is destined for a refactor of docs that plasters warnings all over spare parts, I'm inclined to migrate that content into the more widely used how-to pattern. Will update shortly.